docs/docs/databases/hive.mdx

```diff
@@ -12,5 +12,5 @@ The [pyhive](https://pypi.org/project/PyHive/) library is the recommended way to
 The expected connection string is formatted as follows:
 
 ```
-hive://hive@{hostname}:{port}/{database}
+hive://hive@{hostname}:{port}/{database}?auth={NONE/LDAP/KERBEROS}
```

Comment:
I think that the different auth should use engine parameters rather than URI in SQLAlchemy. for example, [pyhive](https://github.com/dropbox/PyHive#passing-session-configuration) manual.

![image](https://user-images.githubusercontent.com/2016594/171184793-d5408601-7935-4c32-87c8-3ca22408a8f0.png)

Reply:
> 我认为不同的身份验证应该在SQLAlchemy中使用引擎参数而不是URI。例如，[pyhive](https://github.com/dropbox/PyHive#passing-session-configuration) manual。
> 
> ![图像](https://user-images.githubusercontent.com/2016594/171184793-d5408601-7935-4c32-87c8-3ca22408a8f0.png)

Thank you for your answer. For me, as a newly graduated student on the road, my understanding of some knowledge is not comprehensive enough. Your answer let me have a new understanding of this

Reply:
> 万丈高楼平地起. You are welcome!

感谢指导！哈哈哈